### ISSUE TITLE
AutoFlush broken

### ISSUE THREAD

**Issue by dv-petrov** [pmqs/Archive-Zip-SimpleZip#16]
Can't locate object method "_autoflush" via package "IO::Compress::Zip" at C:/Strawberry/perl/site/lib/Archive/Zip/SimpleZip.pm line 249.

**Comment by pmqs** [comment]
reproduce with this one-liner
```
$ perl -MArchive::Zip::SimpleZip -e '$z = new Archive::Zip::SimpleZip(\$out, autoflush=>1) or die "xx"; $z->addString("x", name=>"y"); '
Can't locate object method "_autoflush" via package "IO::Compress::Zip" at /home/paul/perl/install/lib/site_perl/5.30.2/Archive/Zip/SimpleZip.pm line 249.
```

**Comment by pmqs** [comment]
Typo error. Fixed in 310b8c276a969bb073f405c7838f8f98a540ad65
